### ISSUE TITLE
actions issue is blocking the Engine rolls

### ISSUE THREAD

**Issue by flar** [flutter/flutter#129666]
https://github.com/flutter/flutter/actions/runs/5394238608/jobs/9795126021

```
##[debug]  bin/internal/fuchsia-linux.version
Error: Error: found unexpected type for label 'a: accessibility' (should be array of config options)
Error: found unexpected type for label 'a: accessibility' (should be array of config options)
```

**Comment by ricardoamador** [comment]
Okay this has been fixed. Keeping it open and monitoring pull requests for a bit.

**Comment by flar** [comment]
Fixed for the engine roller: https://github.com/flutter/flutter/pull/129678

**Comment by dcharkes** [comment]
> Okay this has been fixed.

What was the issue? I'm seeing something similar on https://github.com/dart-lang/native/pull/78 (with this issue being the only match of the exact error message on Google).

**Comment by ricardoamador** [comment]
Hi @dcharkes this is the most current update for the labeler: https://github.com/flutter/flutter/pull/129793/files, It reverses the change you linked.

**Comment by dcharkes** [comment]
> Hi @dcharkes this is the most current update for the labeler: https://github.com/flutter/flutter/pull/129793/files, It reverses the change you linked. But I will take a look.

Thanks! We ended up reverting the update to the Github labler action on our repo: https://github.com/dart-lang/native/pull/83/files

**Comment by ricardoamador** [comment]
Ah okay. It does look like the same thing happened with a non stable version of the labeler but for us it was updated to a beta version that caused the breakage: https://github.com/flutter/flutter/pull/129673/files
